### ISSUE TITLE
Sporadically no input signal detected from Apple TV after standby

### ISSUE THREAD

**Issue by CiNcH83** [CiNcH83/bravia_atv2#55]
When woken after being in standby for a longer period of time, the BRAVIA often won't detect a signal on the HDMI2 port from an Apple TV 4K, only exhibiting a flashing screen, switching back and forth between HDMI2 and HDMI3/ARC. Only rebooting the BRAVIA fixes the problem.

**Comment by paulozoom** [comment]
I’m on FW V6.6545 and this still happens. Do you have a fix or workaround?

**Comment by CiNcH83** [comment]
No. I already closed the ticket for FW 6.6510 but then re-opened as it happened again. I could not reproduce the issue in weeks. It at least seems to happen much less frequently in my case. Previously I could reproduce it every other day...

**Comment by paulozoom** [comment]
The issue I have is this:

1. I press the Menu button on the Apple TV remote to wake it.
2. The Sony TV lights up with the screen fully black (the backlighting is on though), and then immediately turns off again. No on-screen elements or UI. This repeats if I try again.
3. At this point, the only way to turn the TV on is with the TV remote. But if I do so, the TV goes through a cold boot, showing the Android logo and the loading animation, which takes a while.

This didn’t happen before Oreo. Have you noticed this issue?

**Comment by CiNcH83** [comment]
> The Sony TV lights up with the screen fully black (the backlighting is on though), and then immediately turns off again. No on-screen elements or UI. This repeats if I try again.

> At this point, the only way to turn the TV on is with the TV remote. But if I do so, the TV goes through a cold boot, showing the Android logo and the loading animation, which takes a while.

Sounds different to the issue that I had. In my case, TV went on, an UI showed up in the lower right corner, stating that no signal has been detected on HDMI2. Had to manually reboot the TV in order to fix it.

**Comment by CiNcH83** [comment]
Re-opened. Happened again yesterday.
